### PR TITLE
6084 installer lacking namespaces

### PR DESCRIPTION
**Pull request by baukezwaan**
Fixes for usage of namespaces in the installer #6084 
(and also the right jquery version @dhensby as mentioned in #6100)

### CODE REVIEW

**Comment by dhensby** [comment]
@tractorcow I'd appreciate you looking over this, please

**Comment by tractorcow** [comment]
Ah, I'm sorry I didn't see your pull request; I've re-built installer for 4.0 more significantly over at https://github.com/silverstripe/silverstripe-framework/pull/6167.

File: Dev/Install/install.php5 [hunk 1]
```diff
@@ -1047,7 +1052,7 @@ class InstallRequirements {
 		$adapters = DatabaseAdapterRegistry::get_adapters();
 		if(isset($adapters[$databaseClass])) {
 			$helperPath = $adapters[$databaseClass]['helperPath'];
-			$class = str_replace('.php', '', basename($helperPath));
+			$class = '\SilverStripe\Dev\Install\\' . str_replace('.php', '', basename($helperPath));
```
Comment (tractorcow) [inline Dev/Install/install.php5]: Unfortunately not all helpers will be in this namespace, only the mysql one. :)

**Comment by baukezwaan** [comment]
No problem... it was only my first PR attempt ever, haha!

**Pull request closed without merging**